samples/snippets/src/test/java/com/example/bigtable/ConfigureConnectionPoolTest.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.bigtable;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import org.hamcrest.CoreMatchers;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class ConfigureConnectionPoolTest {
+
+  private static final String INSTANCE_ENV = "BIGTABLE_TESTING_INSTANCE";
+  private static String projectId;
+  private static String instanceId;
+  private ByteArrayOutputStream bout;
+
+  private static String requireEnv(String varName) {
+    String value = System.getenv(varName);
+    assertNotNull(
+        String.format("Environment variable '%s' is required to perform these tests.", varName),
+        value);
+    return value;
+  }
+
+  @BeforeClass
+  public static void beforeClass() {
+    projectId = requireEnv("GOOGLE_CLOUD_PROJECT");
+    instanceId = requireEnv(INSTANCE_ENV);
```

Comment:
I know other tests are already formatted like this, but I think it's clearer to inline this variable since we already do for projectId. It's confusing at first glance if the env variable should be `INSTANCE_ENV` or `BIGTABLE_TESTING_INSTANCE`.

Reply:
fixed